### PR TITLE
109 data description aw

### PR DESCRIPTION
**Pull request by alwil**
I have added description to the `emu_raw` and `emu_metadata` datasets. 
Let me know if there's something to be adapted.

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#' A data frame with metadata of theses from the EMU program.
+#' The data frame can be joined with the `emu_raw` data frame via ID column.
+#'
+#' @format ## `emu_metadata`
+#' A data frame with 83 rows and 8 columns:
+#' \describe{
+#'   \item{ID}{ID of the thesis}
+#'   \item{graduation_year}{Raw text of the thesis.}
+#'   \item{graduation_semester}{Raw text of the thesis.}
+#'   \item{title}{Title of the thesis.}
```
Comment (cforgaci) [inline R/data.R:28]: I just realised that we will need `subtitle` and `full_title` as well. Or at least the `full_title` that was split by the `:` into `title` and `subtitle`. All those three fields are used in `find_meta_stopwords()`.

File: R/data.R [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#'   \item{location}{The location that the thesis is focused on.}
+#'   \item{latitude}{Latitude of the thesis location.}
```
Comment (cforgaci) [inline R/data.R:30]:
```suggestion
#'   \item{latitude}{Latitude of the thesis location geocoded from `location`.}
```

**Review by cforgaci — COMMENTED** [review]
Thanks, @alwil.  See my comments on the columns of `emu_raw`

File: R/data.R (hunk 1 above)
Comment (alwil) [reply R/data.R:29]: Thanks @cforgaci! 

I've incorporated the changes. Please have a loook

Comment (cforgaci) [reply R/data.R:29]: Here I see still `title`. Did you decide to drop `subtitle` and `full_title` or you replaced the values of `title` with `full_title`?

**Review by cforgaci — APPROVED** [review]

**Pull request merged**